CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -988,6 +1037,17 @@ void IWebBrowserInternal.OnAfterBrowserCreated(IBrowser browser)
             {
                 if (!IsDisposed)
                 {
+                    if (_imeWin != null)
+                    {
+                        _imeWin.Dispose();
+                    }
+
+                    if (source != null)
+                    {
+                        _imeWin = new OsrImeWin(source.Handle, this.browser);
```

Comment:
If you create `OsrImeWin` when focus is gained and destroyed it when focus Is lost, is there a reason it's created here?